code/modules/goonhub/event_recording/types/_TracksPlayer.dm

```diff
@@ -0,0 +1,21 @@
+
+ABSTRACT_TYPE(/datum/eventRecordBody/TracksPlayer)
+/// For events that require a player ID - ABSTRACT
+/datum/eventRecordBody/TracksPlayer
+	var/player_id	= null // integer
+
+/datum/eventRecordBody/TracksPlayer/New(
+	player_id
+)
+	. = ..()
+	src.player_id = player_id
+
+/datum/eventRecordBody/TracksPlayer/VerifyIntegrity()
+	if (
+		isnull(src.player_id)
+	)
+		return FALSE
+
+/datum/eventRecordBody/TracksPlayer/ToList()
+	. = ..()
+	. = list("player_id" = src.player_id)
```

Comment:
this won't work like you think it works i don't think?

Reply:
It does work actually